src/main/scala/com/databricks/spark/sql/perf/Benchmarkable.scala

```diff
@@ -45,9 +45,9 @@ trait Benchmarkable extends Logging {
     sparkContext.setJobDescription(s"Execution: $name, $description")
     beforeBenchmark()
     val result = if (forkThread) {
-      runBenchmarkForked(includeBreakdown, description, messages, timeout)
-    } else {
       doBenchmark(includeBreakdown, description, messages)
+    } else {
```

Comment:
You are right. It works on databricks, though, so there must be an error somewhere else.